sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,248 @@
+## APIs:
+class ServiceBusClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        credential : TokenCredential,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = TransportType.Amqp,
+    ):
+
+    def __enter__(self):
+    def __exit__(self, *kwargs):
+    def close(self):
+
+    @classmethod
+    def from_connection_string(cls, conn_str, **kwargs):
+
+    def get_queue_sender(self, queue_name, **kwargs) -> ServiceBusSender:
+    def get_queue_receiver(self, queue_name, **kwargs) -> ServiceBusReceiver:
+    def get_topic_sender(self, topic_name, **kwargs) -> ServiceBusSender:
+    def get_subscription_receiver(self, topic_name, subscription_name, **kwargs) -> ServiceBusReceiver:
+
+class ServiceBusSender:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        credential: TokenCredential,
+        queue_name: str = None,
+        topic_name: str = None,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = TransportType.Amqp,
+        retry_total : int = 3,
+        retry_backoff_factor : float = 30,
+        retry_backoff_maximum : int = 120  # cur_retry_backoff = retry_backoff_factor * (2^cur_retry_time)
+    ) -> None:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        queue_name: str = None,
+        topic_name: str = None,
+        **kwargs
+    ) -> ServiceBusSender:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
+
+    def create_batch(
+        self,
+        max_size_in_bytes : int = None
+    ) -> BatchMessage:
+
+    def send(
+        self,
+        message : Union[Message, BatchMessage],
+        session_id : str = None,
+        message_timeout : float = None
+    ) -> None:
+    def schedule(
+        self,
+        message : Union[Message, BatchMessage],
+        schedule_time_utc : datetime,
+        session_id : str = None
```

Comment:
Hm..do you have a session id parameter on the message object already? Should it be passed in or just extracted?